benchmarks/zlib_zlib_uncompress_fuzzer/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder@sha256:84df55d087d82b65e3a594c01248d59f49d513d4d7362014b417e640317b11f4
+MAINTAINER inferno@chromium.org
+RUN apt-get update && apt-get install -y make autoconf automake libtool
+RUN git clone --depth 1 -b develop https://github.com/madler/zlib.git
+WORKDIR zlib
+COPY build.sh *_fuzzer.c* $SRC/
```

Comment:
nit: can  `*_fuzzer.c*` be replaced with `zlib_uncompress_fuzzer.cc`?

Reply:
It's OK if we ask for this here, but I'm not going to ask external contributors to make this many changes to benchmarks. Just what it is necessary to avoid clogging up our repo. I'm with benchmarks containing some unneeded lines if it means less work for contributors.